src/psyclone/tests/parse/kernel_test.py

```diff
@@ -635,7 +721,7 @@ def test_get_integer_array():
 def test_get_int_array_name_err(monkeypatch):
     ''' Tests that we raise the correct error if there is something wrong
     with the Name in the assignment statement obtained from fparser2. '''
-    from fparser.two import Fortran2003
+
```

Comment:
I would suggest a blank line before the closing quotations instead as in other tests.

Reply:
Done